service/service.py

```diff
@@ -560,6 +611,62 @@ def delete(self, product_id):
 
         return "", status.HTTP_204_NO_CONTENT
 
+######################################################################
```

Comment:
um, why do we need to migrate this endpoint to RESTPlus? we do not use it anymore we can just delete it right?

Reply:
Also I migrate this part for the unitests... This route used a lot in the update tests, so I made it to restplus to match the restplus version of update endpoint.

Reply:
I guess it is okay since you already made a restplus version it, but you should remove the "product1", "product2" arguments.

Reply:
Is that okay if I delete the endpoint of the relationship and refactor all the related unitests? Because I feel it may take same bunch of time to only refactor "product1" and "product2"...

Reply:
Actually can we get on a zoom call real quick?

Reply:
Sure no problem! That will be perfect!

Reply:
I sent a zoom link on slack